packages/workflow/src/Extensions/DateExtensions.ts

```diff
@@ -117,6 +117,10 @@ function extract(inputDate: Date | DateTime, extraArgs: DatePart[]): number | Da
 		return Math.floor(diff / (1000 * 60 * 60 * 24));
 	}
 
+	if (part === 'week') {
+		part = 'weekNumber';
+	}
```

Comment:
I'd add this to `DATETIMEUNIT_MAP` instead.

Reply:
Added but then reverted because this works for `extract` but breaks `startOf`.